File: Free Html To Pdf Converter for ASP.NET MVC/[VB]-Free Html To Pdf Converter for ASP.NET MVC/VB/SelectPdf.Samples/Controllers/ViewHttpCookiesController.vb
Namespace Controllers
    Public Class ViewHttpCookiesController
        Inherits Controller

        ' GET: ViewHttpCookies
        Public Function Index() As ActionResult
            Dim cookies As String = String.Empty
            For Each key As String In Request.Cookies
                Dim cookie As HttpCookie = Request.Cookies(key)
                cookies += "<br/>" + cookie.Name + ": " + cookie.Value
            Next

            ViewData.Add("cookiesValue", cookies)

            Return View()
        End Function
    End Class
End Namespace

File: Free Html To Pdf Converter for ASP.NET MVC/[C#]-Free Html To Pdf Converter for ASP.NET MVC/C#/SelectPdf.Samples/Controllers/ViewHttpCookiesController.cs
using System.Web;
using System.Web.Mvc;

namespace SelectPdf.Samples.Controllers
{
    public class ViewHttpCookiesController : Controller
    {
        // GET: ViewHttpCookies
        public ActionResult Index()
        {
            string cookies = string.Empty;
            foreach (string key in Request.Cookies)
            {
                HttpCookie cookie = Request.Cookies[key];
                cookies += "<br/>" + cookie.Name + ": " + cookie.Value;
            }

            ViewData.Add("cookiesValue", cookies);
            
            return View();
        }
    }
}